src/backend/prisma/schema.prisma

```diff
@@ -53,6 +58,9 @@ model User {
   email                   String           @unique
   emailId                 String?          @unique
   role                    Role             @default(GUEST)
+  userSettingsId          String?          @unique
+  userSettings            UserSettings?    @relation(fields: [userSettingsId], references: [id])
```

Comment:
I think that the user should have just the userSettings object and the userSettings model should be concerned with the id and @relation tag. I don't know if it matterrs but it makes more rsense to me in the sense that